src/modules/Lan.rb

```diff
@@ -348,38 +348,48 @@ def Read(cache)
       return false if Abort()
       # Progress step 4/8
       ProgressNextStage(_("Reading network configuration...")) if @gui
-      NetworkConfig.Read
+      begin
+        NetworkConfig.Read
 
-      @ipv6 = readIPv6
+        @ipv6 = readIPv6
 
-      Builtins.sleep(sl)
+        Builtins.sleep(sl)
 
-      return false if Abort()
-      # Progress step 5/8
-      ProgressNextStage(_("Reading hostname and DNS configuration...")) if @gui
-      DNS.Read
-      Host.Read
-      Builtins.sleep(sl)
+        return false if Abort()
+        # Progress step 5/8
+        ProgressNextStage(_("Reading hostname and DNS configuration...")) if @gui
+        DNS.Read
 
-      return false if Abort()
-      # Progress step 6/8
-      ProgressNextStage(_("Reading installation information...")) if @gui
-      #    ReadInstallInf();
-      Builtins.sleep(sl)
+        Host.Read
+        Builtins.sleep(sl)
 
-      return false if Abort()
-      # Progress step 7/8
-      ProgressNextStage(_("Reading routing configuration...")) if @gui
-      Routing.Read
-      Builtins.sleep(sl)
+        return false if Abort()
+        # Progress step 6/8
+        ProgressNextStage(_("Reading installation information...")) if @gui
+        #    ReadInstallInf();
+        Builtins.sleep(sl)
 
-      return false if Abort()
-      # Progress step 8/8
-      ProgressNextStage(_("Detecting current status...")) if @gui
-      NetworkService.Read
-      Builtins.sleep(sl)
+        return false if Abort()
+        # Progress step 7/8
+        ProgressNextStage(_("Reading routing configuration...")) if @gui
+        Routing.Read
+        Builtins.sleep(sl)
+
+        return false if Abort()
+        # Progress step 8/8
+        ProgressNextStage(_("Detecting current status...")) if @gui
+        NetworkService.Read
+        Builtins.sleep(sl)
+
+        return false if Abort()
+      rescue RuntimeError => error
```

Comment:
same IOError can happen here like if you try to write to nfs that is temporary unavailable